json/src/test/scala/com/gu/contentapi/json/JsonParserEditionsTest.scala

```diff
@@ -1,11 +1,13 @@
 package com.gu.contentapi.json
 
-import com.gu.contentapi.json.utils.JsonLoader.loadJson
+import com.gu.contentapi.client.model.v1.EditionsResponse
+import com.gu.contentapi.json.utils.JsonHelpers._
+import com.gu.contentapi.json.CirceDecoders._
 import org.scalatest.{FlatSpec, Matchers}
 
 class JsonParserEditionsTest extends FlatSpec with Matchers {
```

Comment:
I've deleted the lot, as they don't test anything that isn't covered by the 'round-trip' tests